lib/tools/system-calls.js

```diff
@@ -85,8 +84,17 @@ systemCallMethods.getBinaryFromSdkRoot = _.memoize(async function (binaryName) {
                     path.resolve(this.sdkRoot, "tools", "bin", binaryName)];
   // get subpaths for currently installed build tool directories
   let buildToolDirs = await getDirectories(path.resolve(this.sdkRoot, "build-tools"));
+  const buildDirsByMTime = {};
+  for (const buildToolDir of buildToolDirs) {
+    const fullPath = path.resolve(this.sdkRoot, "build-tools", buildToolDir);
+    const {mtime} = await fs.stat(fullPath);
```

Comment:
`mtime` is now a string with a human readable date.

Reply:
The node doc says it should be a Date object, but if you says so...

Reply:
Ah. `sort` is coercing them into strings, then.